infra/cifuzz/fuzz_target.py

```diff
@@ -367,11 +368,44 @@ def download_latest_corpus(self):
     return download_and_unpack_zip(corpus_url, corpus_dir)
 
 
-def download_and_unpack_zip(http_url, out_dir):
+def download_url(url, filename, num_retries=3):
+  """Downloads the file located at |url|, using HTTP to |filename|.
+
+  Args:
+    url: A url to a file to download.
+    filename: The path the file should be downloaded to.
+    num_retries: The number of times to retry the download on
+       ConnectionResetError.
+
+  Returns:
+    True on success.
+  """
+  sleep_time = 1
+
+  for _ in range(num_retries):
+    try:
+      urllib.request.urlretrieve(url, filename)
+      return True
+    except urllib.error.HTTPError:
+      # In these cases, retrying probably wont work since the error probably
```

Comment:
maybe check error code 404. otherwise 500 or something, we should retry.

Reply:
Hmm...Not sure it's worth the complexity. I think I will leave as is.

Reply:
We do it all over place in ClusterFuzz since 404 is the only one where you need to skip. for 500, it makes sense to retry, e.g. see https://github.com/google/clusterfuzz/blob/master/src/python/google_cloud_utils/storage.py#L1136